clarifai/cli/model.py

```diff
@@ -68,3 +69,130 @@ def run_locally(model_path):
   except Exception as e:
     click.echo(f"Failed to starts model server locally: {e}", err=True)
+
+
+@model.command()
+@click.option(
+    '--config',
+    type=click.Path(exists=True),
+    required=False,
+    help='Path to the model predict config file.')
+@click.option('--model_id', required=False, help='Model ID of the model used to predict.')
+@click.option('--user_id', required=False, help='User ID of the model used to predict.')
+@click.option('--app_id', required=False, help='App ID of the model used to predict.')
+@click.option('--model_url', required=False, help='Model URL of the model used to predict.')
+@click.option('--file_path', required=False, help='File path of file for the model to predict')
+@click.option('--url', required=False, help='URL to the file for the model to predict')
+@click.option('--bytes', required=False, help='Bytes to the file for the model to predict')
+@click.option(
+    '--input_id', required=False, help='Existing input id in the app for the model to predict')
+@click.option('--input_type', required=False, help='Type of input')
+@click.option(
+    '-cc_id',
+    '--compute_cluster_id',
+    required=False,
+    help='Compute Cluster ID to use for the model')
+@click.option('-np_id', '--nodepool_id', required=False, help='Nodepool ID to use for the model')
+@click.option(
+    '-dpl_id', '--deployment_id', required=False, help='Deployment ID to use for the model')
+@click.option(
+    '--inference_params', required=False, default='{}', help='Inference parameters to override')
+@click.option('--output_config', required=False, default='{}', help='Output config to override')
+@click.pass_context
+def predict(ctx, config, model_id, user_id, app_id, model_url, file_path, url, bytes, input_id,
+            input_type, compute_cluster_id, nodepool_id, deployment_id, inference_params,
+            output_config):
+  """Predict using the given model"""
+  import json
+
+  from clarifai.client.deployment import Deployment
+  from clarifai.client.input import Input
+  from clarifai.client.model import Model
+  from clarifai.client.nodepool import Nodepool
+  from clarifai.utils.cli import from_yaml
+  if config:
+    config = from_yaml(config)
+    model_id = config.get('model_id', model_id)
+    user_id = config.get('user_id', user_id)
+    app_id = config.get('app_id', app_id)
+    model_url = config.get('model_url', model_url)
+    file_path = config.get('file_path', file_path)
+    url = config.get('url', url)
+    bytes = config.get('bytes', bytes)
+    input_id = config.get('input_id', input_id)
+    input_type = config.get('input_type', input_type)
+    compute_cluster_id = config.get('compute_cluster_id', compute_cluster_id)
+    nodepool_id = config.get('nodepool_id', nodepool_id)
+    deployment_id = config.get('deployment_id', deployment_id)
+    inference_params = config.get('inference_params', inference_params)
+    output_config = config.get('output_config', output_config)
```

Comment:
WDYT of this suggestion, this is redundant code, so maybe in a one line would be fine.

```suggestion
 model_id, user_id, app_id, model_url, file_path, url, bytes, input_id, input_type, compute_cluster_id, nodepool_id, deployment_id, inference_params, output_config = (config.get(k, v) for k, v in [('model_id', model_id), ('user_id', user_id), ('app_id', app_id), ('model_url', model_url), ('file_path', file_path), ('url', url), ('bytes', bytes), ('input_id', input_id), ('input_type', input_type), ('compute_cluster_id', compute_cluster_id), ('nodepool_id', nodepool_id), ('deployment_id', deployment_id), ('inference_params', inference_params), ('output_config', output_config)]))
```